docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: "3.9"
+services:
+  db:
+    image: postgres
+    restart: always
+    environment: 
+      POSTGRES_PASSWORD: dealbot-admin
+      POSTGRES_USER: admin
+      POSTGRES_DB: dealbot
+  controller:
+    build: .
+    command: ["controller", "--listen", ":8764", "--metrics", "prometheus", "--driver", "postgres", "--dbloc", "postgresql://admin:dealbot-admin@db:5432/dealbot?sslmode=disable"]
+    ports:
+      - "8764:8764"
+    depends_on: 
+      - db
+  dealbot:
+    build: .
+    command: ["mock", "--endpoint", "http://controller:8764"]
```

Comment:
since this is just a testing compose setup, we should probably give it a more appropriate name?

Reply:
sure!